Sources/Extensions/UIButton+ImageDownloader.swift

```diff
@@ -9,11 +9,25 @@
 import UIKit
 
 public extension ObjectWrapper where T: UIButton {
+
+    var imageUrl: URL? {
+        get {
+            imageDownloaderReceipt?.url
+        }
+        set {
+            setImageUrl(url: newValue)
+        }
+    }
     
-    func setImage(with url: URL,
-                  for state: UIControl.State = .normal,
-                  imageDownloader: ImageDownloading = ImageDownloader.shared,
-                  completionHandler: ImageDownloader.CompletionHandler? = nil) {
+    func setImageUrl(url: URL?,
```

Comment:
make it either `setImage(url: URL` or `setImageURL(_ url: URL)

Reply:
ok done